DependencyInjection/WebstackUserExtension.php

```diff
@@ -31,6 +31,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('webstack_user.model.user.class', $config['user_class']);
         $container->setParameter('webstack_user.model.user.class.email_as_username', $config['use_email_as_username']);
 
+        if (isset($config['password'])) {
+            $container->setParameter('webstack_user.security.password.min_strength', $config['password']['min_strenght']);
+            $container->setParameter('webstack_user.security.password.min_length', $config['password']['min_lenght']);
```

Comment:
```suggestion
            $container->setParameter('webstack_user.security.password.min_strength', $config['password']['min_strentht']);
            $container->setParameter('webstack_user.security.password.min_length', $config['password']['min_length']);
```

Reply:
Hier stond een typo in, heb hem gecorrigeerd in de suggestion